src/components/environment-banner.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ *  A banner to conditionally display warnings to users about the environment.
+ */
+
+import React from 'react';
+
+export default class EnvironmentBanner extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      apiEnv: 'production',
```

Comment:
Would you mind unabbreviating this? We try and avoid any abbreviations that aren’t used in normal speech.